.github/workflows/test.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     - uses: actions/checkout@v3
     - uses: actions/setup-python@v3
       with:
-        python-version: "3.7"
+        python-version: "3.13"
```

Comment:
Why do we need this change?

Reply:
Decided to  bump  based on this error https://github.com/stackrox/workflow/actions/runs/14303872268/job/40083306555?pr=111